tests/functional/amo_sign_up.js

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'require',
+  'tests/functional/lib/helpers',
+  'tests/functional/lib/fx-desktop'
+], function (intern, registerSuite, require, FunctionalHelpers, FxDesktopHelpers) {
+  var config = intern.config;
+  var PAGE_URL = config.fxaContentRoot + 'signin?context=fx_desktop_v1';
```

Comment:
A couple of problems here:

* Use `signup` instead of `signin`
* AMO is going to use the OAuth flow instead of the Sync flow, so no `context` will be specified. Instead they'll open something like `signup?client_id=amo_client_id&state=state`